compiler/il/OMRILOpCodesEnum.hpp

```diff
@@ -534,6 +534,7 @@
    vcalli,     // indirect call returning a vector
    vternary,   // vector ternary operator
    v2v,        // vector to vector conversion. preserves bit pattern (noop), only changes datatype
+   vl2vd,      // vector to vector conversion. converts each long element to double
    vconst,     // vector constant
```

Comment:
Do we expect vi2vf (vector integer to vector single precision floating point) or opCodes for other combinations of data types in the future?  I am wondering if we can treat conversions similarly to vadd/vsub etc, element data type being kept somewhere else.  I know there are two types involved in conversions, different from vadd/vsub etc.